frappe/public/js/frappe/model/perm.js

```diff
@@ -16,7 +16,7 @@ Object.assign(window, {
 
 $.extend(frappe.perm, {
 	rights: ["select", "read", "write", "create", "delete", "submit", "cancel", "amend",
-		"report", "import", "export", "print", "email", "share", "set_user_permissions"],
+		"report", "import", "export", "print", "email", "share", "set_user_permissions", "if_owner"],
```

Comment:
not sure if adding this in `rights` is the right decision:
- it's not really a right 😄 
- it's absent in `frappe.permissions.rights`

Reply:
Added it here, so that line `115` will make it available for line `69`. Where would you prefer?

Reply:
Perhaps we should add inside `get_role_permissions` itself, after the loop over `rights`.

Note that for this to work correctly, we will need to ensure that `if_owner` only applies if _all_ permitted roles have `if_owner` checked. In the counterpart in `frappe.permissions`, you'll find the use of `has_permission_without_if_owner_enabled` that accomplishes this.